front50-core/src/main/groovy/com/netflix/spinnaker/front50/listeners/ChaosMonkeyApplicationPermissionEventListener.java

```diff
@@ -69,11 +69,7 @@ public boolean supports(ApplicationPermissionEventListener.Type type) {
 
     Application application = applicationDAO.findByName(updatedPermission.getName());
 
-    if (isChaosMonkeyEnabled(application)) {
-      applyNewPermissions(updatedPermission, true);
-    } else {
-      applyNewPermissions(updatedPermission, false);
-    }
+    applyNewPermissions(updatedPermission, isChaosMonkeyEnabled(application));
```

Comment:
nitpick: would it make sense to have `applyNewPermissions` be responsible for calling `isChaosMonkeyEnabled`?

Reply:
Yeah, if I get back to this I'll make that happen.